docs/user/ppl/admin/connectors/s3glue_connector.rst

```diff
@@ -42,7 +42,9 @@ Glue Connector Properties.
         * Basic Auth required ``glue.indexstore.opensearch.auth.username`` and ``glue.indexstore.opensearch.auth.password``
         * AWSSigV4 Auth requires ``glue.indexstore.opensearch.auth.region``  and ``glue.auth.role_arn``
     * ``glue.indexstore.opensearch.region`` [Required for awssigv4 auth]
-* ``glue.lakeformation.enabled`` determines whether to enable lakeformation for queries. Default value is ``"false"`` if not specified
+* ``glue.iceberg.enabled`` determines whether to enable Iceberg for the session. Default value is ``"false"`` if not specified.
+* ``glue.lakeformation.enabled`` determines whether to enable Lake Formation for queries when Iceberg is also enabled. If Iceberg is not enabled, then this property has no effect. Default value is ``"false"`` if not specified.
+* ``glue.lakeformation.session_tag`` what session tag to use when assuming the data source role. This property is required when both Iceberg and Lake Formation are enabled.
```

Comment:
Have we introduced validations on these conditions while creating Glue datasource?

Reply:
Thanks for catching this! I've added them now.